eras/mary/impl/CHANGELOG.md

```diff
@@ -1,9 +1,13 @@
 # Version history for `cardano-ledger-mary`
 
-## 1.7.0.2
+## 1.7.1.0
 
 *
 
+### `testlib`
+
+* Switch to using `ImpSpec` package
+
 ## 1.7.0.1
 
 *
```

Comment:
`1.7.0.1` have been released to CHaP, so removing the whole section would be wrong.

The fact that there is an empty bullet point in that section is not a problem, quite the opposite, it serves as a good indicator that there were no notable changes in that release. In fact if we follow our release process we can't remove such empty bullet points, because that implies a PR before making a release, which we do not require.
We could remove it after the release, but that would make little difference, since released version of the package would still have this empty bullet point in its changelog.

So, I don't think we need to worry about this.

Reply:
Thanks for the explanation. That makes sense. I had been wondering why we made a release when there was nothing to note in the changelog, but I was forgetting that the changelog is for externally-visible changes only.